sr/comp/scorer/app.py

```diff
@@ -63,6 +63,9 @@ def is_match_done(match):
 
 
 def form_to_score(match, form):
+    def get_tokens(key):
+        tokens_str = form.get(key, '')
```

Comment:
Would this look nicer as `return int(form.get(key, 0))`, as it's locally what you're trying to achieve, I think?

Reply:
Unfortunately not, that errors given the input `form = {key: ''}`.

Reply:
aaah, of course!